### PR TITLE
Fix duplicate TEST_DATA

### LINKED ISSUE #36: TEST_DATA defined twice
I (@yftacherzog) asked for TEST_DATA to be reused between the tests and the migration files, and @thetaxas2468 did that, defining the TEST_DATA inside the tests, which @Omeramsc later said should not be done.

I still wouldn't like it to be defined twice, as it makes the tests more fragile given the coupling between the migration data and the tests content. I also agree that it should be defined in the migration file.

If no objections, I think the proper way to have it is to define it on the migration files, and then import it on the tests. Generally speaking, we prefer having the tests import constants coming from the code being tested rather than the other way around.

### PR DESCRIPTION
**Pull request by thetaxas2468**
Here in this commit we are fixing the duplicate test_data variable
that appears once in 0002 test migrate file and once in test_product so
we have it included in the 0002_test_data file and then importing it
every where we want and needed it.
- fixed #36

### CODE REVIEW

File: product/tests/file_for_tests_migrations.py
Comment (yftacherzog) [inline product/tests/file_for_tests_migrations.py:1]: I don't think it's a good idea to have data defined under `tests` and then use it elsewhere (e.g. code or migrations).

As i mentioned before, I think it would be best to have it defined inside the migrations file and then imported on the test file.

As this has been going back and forth now for a few times with suggestions from @Omeramsc and myself, I think it will be best to wait for more feedback before changing things again (mainly, to avoid unnecessary work on your side).

Comment (Omeramsc) [reply product/tests/file_for_tests_migrations.py:1]: this is exactly what I asked to be done on the other patch...

Comment (thetaxas2468) [reply product/tests/file_for_tests_migrations.py]: Fixed!

**Comment by Omeramsc** [comment]
I don't understand the current solution...
There is no need for an extra file (or directory). Just declare the data in the migration file instead of the test file, then you'll be able to import it. 

also please work on the file namings - 
- `constant_file` is a directory, so why is it called a file? that's very confusing.
- `file_for_test_migrations` - "file for" is redundant, of course that's a file. but then `test_migration` is misleading, right? well, that's a hint for the fact this file is not needed in the first place.

**Comment by thetaxas2468** [comment]
> I don't understand the current solution...
> There is no need for an extra file (or directory). Just declare the data in the migration file instead of the test file, then you'll be able to import it. 
> 
> also please work on the file namings - 
> - `constant_file` is a directory, so why is it called a file? that's very confusing.
> - `file_for_test_migrations` - "file for" is redundant, of course that's a file. but then `test_migration` is misleading, right? well, that's a hint for the fact this file is not needed in the first place.

 1-tried many times to import from but i cant due to the migration file names
2-created a dir so maybe it can have more files in the future 
3-meant file for tests and migrations

**Comment by thetaxas2468** [comment]
I have fixed the problem implementing numerical start of a file,and updated few things

**Comment by yftacherzog** [comment]
Just fix the commit message, and I think we're good.

**Comment by yftacherzog** [comment]
lgtm. I leave it to @Omeramsc to merge once he's happy.

**Review by Omeramsc — APPROVED** [review]
LGTM

**Pull request merged**
